src/Microsoft.Health.Dicom.Core/Features/Telemetry/StoreMeter.cs

```diff
@@ -0,0 +1,28 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Health.Dicom.Core.Features.Telemetry;
+
+public sealed class StoreMeter : IDisposable
+{
+    private readonly Meter _meter;
+
+    public StoreMeter()
+    {
+        _meter = new Meter($"{OpenTelemetryLabels.BaseMeterName}.Store", "1.0");
```

Comment:
Same comments about the units and descriptions as in DICOMcast

Reply:
Done